### ISSUE TITLE
constant crashing after opening the wp app 3 times

### ISSUE THREAD

**Issue by gofirst2220** [ferasm/reddhub#328]
Like the title states, it kept crashing after I open it 3 times on my 920 on W10 for phone (latest Fast Ring build).

**Comment by JDOfTheNorth** [comment]
I am unable to repro this.  Does this still repro for you on latest Reddhub & latest windows build?

**Comment by gofirst2220** [comment]
Yep. It does it since I have long list of subreddits (over thousands) and I'm on latest Windows 10 for Phone build (10512 at the time). The PC version works fine since W10Pro 10142 to the newest one ( 10532)
